inc/osvr/Client/DisplayInput.h

```diff
@@ -0,0 +1,63 @@
+/** @file
+    @brief Header
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_DisplayInput_h_GUID_66C8D271_F035_499C_6BD9_DCA114E2C9AF
+#define INCLUDED_DisplayInput_h_GUID_66C8D271_F035_499C_6BD9_DCA114E2C9AF
+
+// Internal Includes
+// - none
+
+// Library/third-party includes
+// - none
+
+// Standard includes
+// - none
+
+namespace osvr {
+namespace client {
+    class DisplayConfigFactory;
+    class DisplayInput {
+      public:
+        DisplayInput(DisplayInput const &) = delete;
```

Comment:
For now, you can remove this line (deleting copy constructor), the deleting-copy-assignment, and the manual implementation of a move-constructor - nothing here is noncopyable.